core/src/test/java/io/grpc/ClientInterceptorsTest.java

```diff
@@ -351,6 +352,33 @@ protected void checkedStart(ClientCall.Listener<RespT> responseListener, Metadat
     ArgumentCaptor<Status> captor = ArgumentCaptor.forClass(Status.class);
     verify(listener).onClose(captor.capture(), any(Metadata.class));
     assertSame(error, captor.getValue().getCause());
+
+    // Make sure nothing bad happens after the exception.
+    ClientCall<?, ?> noop = ((CheckedForwardingClientCall<?, ?>)interceptedCall).delegate();
```

Comment:
@ejona86: hmm, I had looked at previous runs on coveralls, and noticed the lines not being covered, and then the tests running locally and showing them covered.  I was going to point to the coveralls run for this PR, but https://coveralls.io/builds/3840457 for ClientInterceptors is missing.  :/

I added the other methods to the Noop call to have them in one place (hopefully /the/ place to check) in case some test accidentally stops touching those lines in the future.  The use of verifyNoMoreInteractions is kind of moot IMO, the point isn't so much that it isn't invoked, but that any interaction with it is safe and inert.  Making NOOP_CALL public and moving the lines in this PR to a new methods is also fine.


@madongfly I had been trying to minimize the amount of call indirection.  Directly grabbing the delegate avoids involving the parent class.